src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ContentServiceTests.cs

```diff
@@ -169,17 +172,317 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var service = SetupContentService(contentDbContext: contentDbContext);
+
+                var result = await service.AddComment(
+                    release.Id,
+                    release.Content[0].ContentSection.Id,
+                    release.Content[0].ContentSection.Content[0].Id,
+                    new CommentSaveRequest { Content = "New comment" });
+
+                var viewModel = result.AssertRight();
+                Assert.Equal("New comment", viewModel.Content);
+                Assert.Null(viewModel.Resolved);
+
+                var comments = contentDbContext.Comment.ToList();
+                Assert.Equal(2, comments.Count);
+                Assert.Equal("Existing comment", comments[0].Content);
+                Assert.Equal("New comment", comments[1].Content);
+            }
+        }
+
+        [Fact]
+        public async Task ResolveComment_True()
+        {
+            var comment = new Comment
+            {
+                Content = "Existing comment",
+                CreatedById = Guid.NewGuid(),
+            };
+            var release = new Release
+            {
+                Content = new List<ReleaseContentSection>
+                {
+                    new ReleaseContentSection
+                    {
+                        ContentSection = new ContentSection
+                        {
+                            Content = new List<ContentBlock>
+                            {
+                                new HtmlBlock
+                                {
+                                    Comments = new List<Comment>
+                                    {
+                                        comment
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var userId = Guid.NewGuid();
+                var service = SetupContentService(
+                    contentDbContext: contentDbContext,
+                    userId: userId);
+                var result = await service.ResolveComment(comment.Id, true);
```

Comment:
Could make it a little clearer what's going on by using the named parameter i.e. `resolve: true`

Raw boolean arguments usually suck because they tell you very little about what they're doing